web/locales/de.json

```diff
@@ -0,0 +1,70 @@
+{
+    "panel-initial-text": {
+        "youareseeing": "Du siehst jetzt eine",
+        "limitedversion": "limitierte Mobilversion",
+        "forthe": "Für eine",
+        "fullversion": "uneingeschränkte Version",
+        "visitoncomputer": "besuch diese Seite auf Deinem Computer",
+
+        "thisshowsrealtime": "Hier siehst Du in Echtzeit",
+        "whereelectricitycomesfrom": "wo Deine Elektrizität herkommt",
+        "and": "und",
+        "howmuchco2": "wieviel CO2",
+        "wasemitted": "emittiert wurde um sie zu produzieren",
+
+        "takeinaccount": "Wir berücksichtigen Elektrizitäts",
+        "importexport": "-Imports und -Exports",
+        "betweencountries": "zwischen Ländern",
+
+        "thisproject": "Dieses Projekt ist",
+        "opensource": "Open Source",
+        "see": "siehe",
+        "datasources": "Datenquellen",
+
+        "tip": "Tipp: Klicke auf ein Land um zu beginnen"
+    },
+    "country-panel": {
+        "back": "zurück",
+        "source": "Quelle",
+        "fossilfuels": "fossile Brennstoffe",
+        "electricityprice": "Elektrizitätspreis (day-ahead)",
+        "electricityproduction": "Elektrizitätsproduktion",
+        "showemissions": "zeige Emissionen",
+        "bysource": "nach Quelle",
+        "emissions": "Emissionen",
+        "showelectricity": "zeige Elektrizität",
+        "co2signal": "Unser <a href=\"http://www.co2signal.com\" target=\"blank\">CO2 Signal</a> hilft Geräten und elektrischen Fahrzeugen Elektrizität zur richtigen Zeit zu konsumieren, insbesondere bei Grenzübertritt."
```

Comment:
Does that literally mean: it helps especially when you are travelling across borders? That's the way I translated it at least.